docs/src/main/paradox/typed/testing-async.md

```diff
@@ -134,9 +134,13 @@ a dependency on ScalaTest to use this.
 Scala
 :  @@snip [AsyncTestingExampleSpec.scala](/actor-testkit-typed/src/test/scala/docs/org/apache/pekko/actor/testkit/typed/scaladsl/ScalaTestIntegrationExampleSpec.scala) { #scalatest-integration }
 
-Java
+Java 
 :  @@snip [AsyncTestingExampleTest.java](/actor-testkit-typed/src/test/java/jdocs/org/apache/pekko/actor/testkit/typed/javadsl/JunitIntegrationExampleTest.java) { #junit-integration }
 
+Java Junit5
```

Comment:
* this won't work well - in UI you choose Scala or Java
* have a look where it says Scala in this screenshot

<img width="335" alt="image" src="https://github.com/apache/incubator-pekko/assets/11783444/bf5f606c-2cdc-4e36-aa9f-4ed57f67ff74">

Can you remove the 'Java Junit5' and the related `@@snip`? We can have a think about how to add this to the docs later.

Possibly the simplest short term solution is to add a comment to the JunitIntegrationExampleTest with a URL to the GItHub source of the Junit5IntegrationExampleTest.